src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -30,16 +30,16 @@
  * Checks that all package annotations are in the package-info.java file.
  * </p>
  * <p>
- * According to the Java Language Specification.
+ * For Java SE8 and above, placement of package annotations in the package-info.java
+ * file is enforced by the compiler and this check is not necessary.
  * </p>
  * <p>
- * The JLS does not enforce the placement of package annotations.
- * This placement may vary based on implementation. The JLS
- * does highly recommend that all package annotations are
- * placed in the package-info.java file.
- *
- * See <a
- * href="https://docs.oracle.com/javase/specs/jls/se11/html/jls-7.html#jls-7.4.1">
+ * For Java SE7 and below, the Java Language Specification highly recommends
+ * but doesn't require that annotations are placed in the package-info.java file,
+ * and this check can help enforce that placement.
```

Comment:
`can help enforce` I think it should be `can help to enforce`

Reply:
Done